### DISCUSSION TITLE
How well does i3 work with multiple monitors?

### DISCUSSION THREAD

**Discussion by JohnAtl** [i3/i3#5636] [Q&A]
It seems most YouTubers demo in a vm, or with one screen. What's the experience like with, say, three monitors?
Any videos I may have missed?
Thanks

**Comment by derisyan-code** [comment]
I have a 2 monitor setup and other than running a quick exec xrandr command to align a monitor to the right instead of the default, it works flawlessly

**Reply by BirdeeHub** [reply]
https://github.com/BirdeeHub/i3autoXrandrMemory

I made a thing that helps if you unplug and plug in monitors regularly to auto swap your workspaces back onto the monitor and run xrandr config on it

**Reply by BirdeeHub** [reply]
theres also a script in there to just move a workspace to next monitor on keypress. The repo is just 2 bash scripts lol although one is way more complex than the other

**Comment by okraits** [comment]
i3 has very good multimonitor support. Note: a workspace can only be displayed on one output at a time and is bound to that output. You can move it to another output, though.

**Comment by mangosherbet** [comment]
very good.  have xrandr script automatically startup on login - haven't ever had an issue.

**Reply by BirdeeHub** [reply]
https://github.com/BirdeeHub/birdeeSystems/blob/main/common/i3MonMemory/default.nix

I turned it into a nix module :)
